src/components/Filter/util/__tests__/filterUtil.test.js

```diff
@@ -3,7 +3,7 @@ import {
     getDefaultJsonViewValue,
     getFilterAppliedCount,
     getUrlFromList,
-    areSetsEqual
+    areSetsEqual, convertToFilterDropdownOptions
```

Comment:
nit: add on new line for easier readability.

Reply:
done